### DISCUSSION TITLE
Literally nothing works

### DISCUSSION THREAD

**Discussion by Urfajojo** [end-4/dots-hyprland#731] [General]
I installed it using the script and nothing works, no fancy bars, no apps, no cursor, just a plain background and a terminal. I reinstalled it couple times, manually installed the arch-packages and I give up. I would screenshot it if it worked

**Comment by NextStep-IM** [comment]
Did you make an Issue? If not please do so and with the requested info.

**Reply by 0pl1v10n** [reply]
i think he forgot to rename the hyprland.conf.new to hyprland.conf

**Comment by PharmaRacist** [comment]
try running ags manually in the terminal and rename the hyprland.conf.new to hyprland.conf
